app/src/androidTest/java/org/mozilla/focus/activity/CustomTabTest.kt

```diff
@@ -124,10 +125,15 @@ class CustomTabTest {
 
     @Suppress("Deprecation")
     private fun createCustomTabIntent(pageUrl: String): Intent {
+        val pendingIntentFlags = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
nit: We could have this as maybe an extension to encapsulate this behavior and reduce the amount of duplicated code and documentation.
Something like
```
val PendingIntent.defaultFlags
    get() = if (Build.VERSION.SDK_INT > Build.VERSION_CODES.M) {
        PendingIntent.FLAG_IMMUTABLE
    } else {
        0 // No flags. Default behavior.
    }
```